src/components/apphost.js

```diff
@@ -1,456 +1,441 @@
-define(['appSettings', 'browser', 'events', 'htmlMediaHelper', 'webSettings', 'globalize'], function (appSettings, browser, events, htmlMediaHelper, webSettings, globalize) {
-    'use strict';
+import appSettings from 'appSettings';
+import browser from 'browser';
+import events from 'events';
+import htmlMediaHelper from 'htmlMediaHelper';
```

Comment:
```suggestion
import * as htmlMediaHelper from 'htmlMediaHelper';
```

otherwise this breaks playing videos